test/starknet/vanilla_space.ts

```diff
@@ -48,31 +49,59 @@ async function setup() {
   };
 }
 
+function setup_calldata(
+  proposer_address: bigint,
+  execution_hash: bigint,
+  metadata_uri: Array<bigint>,
+  eth_block_number: bigint,
+  params: Array<bigint>
+): Array<bigint> {
+  const calldata = [proposer_address, execution_hash];
+  calldata.push(BigInt(metadata_uri.length));
+
+  for (let i = 0; i < metadata_uri.length; i++) {
+    calldata.push(metadata_uri[i]);
+  }
+
+  calldata.push(eth_block_number);
+  calldata.push(BigInt(params.length));
+
+  for (let i = 0; i < params.length; i++) {
+    calldata.push(params[i]);
+  }
+
+  return calldata;
+}
+
 describe('Space testing', () => {
   it('Simple Vote', async () => {
     console.log('Setup...');
     const { vanillaSpace, vanillaAuthenticator, vanillaVotingStrategy } = await setup();
     const space_contract = BigInt(vanillaSpace.address);
     const execution_hash = BigInt(1);
-    const metadata_uri = BigInt(2);
+    const metadata_uri = strToShortStrArr(
+      'Hello and welcome to Snapshot X. This is the future of governance.'
+    );
+    console.log(metadata_uri);
     const proposer_address = VITALIK_ADDRESS;
     const proposal_id = 1;
-    const params: Array<BigInt> = [];
+    const params: Array<bigint> = [];
     const eth_block_number = BigInt(1337);
+    const calldata = setup_calldata(
+      proposer_address,
+      execution_hash,
+      metadata_uri,
+      eth_block_number,
+      params
+    );
 
     // -- Creates the proposal --
     {
       console.log('Creating proposal...');
       await vanillaAuthenticator.invoke(EXECUTE_METHOD, {
         to: space_contract,
         function_selector: BigInt(getSelectorFromName(PROPOSAL_METHOD)),
-        calldata: [
-          proposer_address,
-          execution_hash,
-          metadata_uri,
-          eth_block_number,
-          BigInt(params.length),
-        ],
+        calldata: calldata,
```

Comment:
```suggestion
        calldata,
```

Reply:
[use camelcase; rename setup_calldata to setupProposeCalldata](https://github.com/snapshot-labs/sx-core/pull/54/commits/9a1ad38aecf21c32382abf5a53c2387beff40eb2)